src/signals/incident-management/components/FilterForm/styled.tsx

```diff
@@ -52,8 +51,12 @@ export const Fieldset = styled.fieldset<{ isSection?: boolean }>`
     width: 1px;
   }
 
-  .invoer + .Label {
-    margin-top: 20px;
+  .invoer {
+    margin-bottom: ${themeSpacing(8)};
+
+    &.Label {
+      margin-top: 20px;
```

Comment:
margins we are using theme spacing for right?

Reply:
Old code didn't touch it. But will change.